peachjam/views/documents.py

```diff
@@ -57,8 +48,15 @@ class DocumentSourceView(DetailView):
 
     def render_to_response(self, context, **response_kwargs):
         if hasattr(self.object, "source_file") and self.object.source_file.file:
-            file = self.object.source_file.file.open()
-            return view_attachment(file)
+            source_file = self.object.source_file
+            file = source_file.file.open()
+            bytes = file.read()
+            response = HttpResponse(bytes, content_type=source_file.mimetype)
```

Comment:
Assigning to a builtin variable. 
```suggestion
            file_bytes = file.read()
            response = HttpResponse(file_bytes, content_type=source_file.mimetype)
```